auter.yumdnfModule

```diff
@@ -95,10 +99,17 @@ function prepare_updates() {
 }
 
 function apply_updates() {
+  # Default RC to 0 which means no updates are available
+  local -i RC=0
+  local APPLYOUTPUT
+  local UPDATEACTION
+  local HISTORY_BEFORE
+  local HISTORY_AFTER
+  local TRANSACTIONID
```

Comment:
These are not entirely required but declaring them here does make it much easier to see what variables are being used in the function so I am ok with that however as mentioned in a previous comment, For consistency, these should be declared using ```declare -l```